rest/route/host_create_test.go

```diff
@@ -286,6 +286,96 @@ func TestMakeHost(t *testing.T) {
 	assert.Equal(ec2Settings2.AMI, "ami-987654")
 }
 
+func TestHostCreateHandler(t *testing.T) {
+	ctx, cancel := context.WithTimeout(context.Background(), 30*time.Second)
+	defer cancel()
+
+	assert := assert.New(t)
+	require := require.New(t)
+	require.NoError(db.ClearCollections(distro.Collection, host.Collection, task.Collection))
+
+	env := &mock.Environment{}
+	assert.NoError(env.Configure(ctx))
+	var err error
+	env.RemoteGroup, err = queue.NewLocalQueueGroup(ctx, queue.LocalQueueGroupOptions{
+		DefaultQueue: queue.LocalQueueOptions{Constructor: func(context.Context) (amboy.Queue, error) {
+			return queue.NewLocalLimitedSize(2, 1048), nil
+		}}})
```

Comment:
I don't think this queue group needs to be initialized since this test doesn't rely on an Amboy queue (it just inserts host documents).